client/blocks/product-selector/README.md

```diff
@@ -51,6 +51,15 @@ The following props can be passed to the Product Selector block:
 			monthly: [ 'jetpack_backup_daily_monthly', 'jetpack_backup_realtime_monthly' ],
 		}
 		```
+	* `optionDescriptions`: ( object ) Optional descriptions of the product options. Each key is a product slug, and the value is the corresponding copy. Example:
```

Comment:
Maybe let's clarify that these are used only when an option has already been purchased and is associated with an existing purchase. Also, let's clarify where these descriptions are displayed (that they replace the default product description)